<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTUIPreferencePageControls</class>
 <widget class="QWidget" name="RTUIPreferencePageControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Handling of unkown prescribed dose:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinRelativeToMax">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="value">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioDefault">
        <property name="text">
         <string>set to default value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioRelativeToMax">
        <property name="text">
         <string>set relative to max dose:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinDefault">
        <property name="suffix">
         <string> Gy</string>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
